schemas/pioneer/heatmap/heatmap.1.parquetmr.txt

```diff
@@ -0,0 +1,13 @@
+message pioneer_heatmap {
+  required binary user (UTF8);
+  required group sessions (LIST) {
+    repeated group list {
+      required group element {
+        optional int64 duration;
+        required int64 start_time;
+        optional binary tab_id (UTF8);
+        required binary url (UTF8);
+      }
+    }
+  }
+}
```

Comment:
Note: I recommend including all metadata, since we can't do schema evolution on `struct`s, so we would have to backfill to add more fields.